.storybook/viewport.js

```diff
@@ -80,39 +109,61 @@ export default {
     name: 'Apple iPad Mini 7.9"',
     styles: {
       width: '768px',
-      height: '1024px'
+      height: '1024px',
+      borderTopWidth: '90px',
+      borderBottomWidth: '90px',
+      borderLeftWidth: '20px',
+      borderRightWidth: '20px',
+      borderRadius: '55px'     
     },
     type: 'tablet'
   },
   iPad: {
     name: 'Apple iPad 10.2"',
     styles: {
       width: '810px',
-      height: '1080px'
+      height: '1080px',
+      borderTopWidth: '85px',
+      borderBottomWidth: '85px',
+      borderLeftWidth: '20px',
+      borderRightWidth: '20px',
+      borderRadius: '45px'
+
```

Comment:
Should have caught this before but please remove this blank line here.

Reply:
Alright!

WIll do. Slipped through my eyes as well.